README.md

```diff
@@ -179,10 +180,10 @@ Export all upmaps for the given OSD spec(s) in a json format usable by import-ma
 Note that the mappings exported will be just the portions of the upmap items pertaining to the selected OSDs (i.e. if a given OSD is the From or To of the mapping), unless `--whole-pg` is specified.
 
 ```
-$ ./pgremapper export-mappings <osdspec> ... [--output <file>] [--whole-pg]
+$ ./pgremapper export-mappings <osdspec> [<osdspec> ...] [--output <file>] [--whole-pg]
```

Comment:
Does the code docs need an update here too?

I'll stop being a pain now :)

Reply:
haha no worries
https://github.com/digitalocean/pgremapper/blob/7928d2f918ff9de9a79b7c0335113de5ac28c9f2/main.go#L306-L310
It LGTM